components/purchase_modal/purchase_modal.tsx

```diff
@@ -281,6 +287,28 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
         );
     }
 
+    learnMoreLink = () => {
+        return (
+            <a
+                className='footer-text'
+                onClick={() => {
+                    trackEvent(
+                        TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+                        'learn_more_prorated_payment',
+                    );
+                }}
+                href={CloudLinks.PRORATED_PAYMENT}
```

Comment:
Looks like this leads to a dead link

Reply:
it is still dead, but it was the link provided, eventually it will work.